lib/app.js

```diff
@@ -2,10 +2,66 @@ const net = require('net');
 const parseRequest = require('./utils/parseRequest');
 const createResponse = require('./utils/createResponse');
 
+const endpointHandlers = [];
```

Comment:
Oops, that was supposed to be an empty object. I'm kind of surprised this works.